blinkstick/blinkstick.py

```diff
@@ -207,16 +208,17 @@ def __init__(self, device=None, error_reporting=True):
                 self.open_device(device)
 
             self.bs_serial = self.get_serial()
+            self.mode = self.get_mode()
 
-    def _usb_get_string(self, device, index):
+    def _usb_get_string(self, device, length, index):
```

Comment:
My USB library seemed to require a length parameter which required a lot of changes like this. Not sure if this is appropriate for master. I could revert.